pkgs/tools/networking/firefoxpwa/default.nix

```
@@ -0,0 +1,90 @@
+# <https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=firefox-pwa>
+{ stdenv
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, symlinkJoin
+, buildFHSUserEnv
+, pkg-config
+, lib
+}:
+let
+  version = "2.4.1";
+  dir = "native";
+  source = fetchFromGitHub {
+    owner = "filips123";
+    repo = "PWAsForFirefox";
+    rev = "v${version}";
+    sha256 = "sha256-AkCTfbWsAF6N1hmowK33XzI2bnBlbzML9IiGjujc8N8=";
+    sparseCheckout = [ dir ];
+  };
+  pname = "firefoxpwa";
+
+  unwrapped = rustPlatform.buildRustPackage {
+    inherit version;
+    pname = "${pname}-unwrapped";
+
+    src = "${source}/${dir}";
+    cargoSha256 = "sha256-BHEFBrDrSZv9Bwl4q+MQA1BAoaJOxsNc/IK6ygrjwVY=";
+
+    doCheck = false;
+
+    nativeBuildInputs = [ pkg-config ];
+    buildInputs = [ openssl.dev openssl ];
+
+    preConfigure = ''
+      # replace the version number in the manifest
+      sed -i 's;version = "0.0.0";version = "${version}";' Cargo.toml
+      # replace the version in the lockfile, otherwise Nix complains
+      sed -zi 's;name = "firefoxpwa"\nversion = "0.0.0";name = "firefoxpwa"\nversion = "${version}";' Cargo.lock
+      # replace the version number in the profile template files
```

```suggestion
      sed -zi 's;name = "firefoxpwa"\nversion = "0.0.0";name = "firefoxpwa"\nversion = "${version}";' Cargo.lock
```
comments should explain the reasons behind a line, not what it is doing and the first comment is obvious.